style.css

```diff
@@ -296,7 +746,7 @@ body {
   }
 
   .form_section {
-    height: 100%;
+    height: 80%;
```

Comment:
- [ ] I have noticed that this height you have added, has caused the form to have a larger space on top of the form heading 😞 . This makes the portfolio mismatch the Figma design. Kindly ensure to fix this height so that your design matches the Figma design.
- [ ] [OPTIONAL] Since we had a mobile-first version of the portfolio. It's always better to maintain the use of `min-width` instead of `max-width`. This will help us make the portfolio responsive easily 👍 .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/220855068-b7988d57-3a77-4400-b429-05afc0a080c0.png)